### PR TITLE
OWNERS: init

### PR DESCRIPTION
**Pull request by hexfusion**
Signed-off-by: Sam Batschelet <sbatsche@redhat.com>

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -0,0 +1,8 @@
+approvers:
+- abhinavdahiya
+- crawford
+- csrwng
+- hexfusion
+- smarterclayton
+- wking
+- rithujohn191
```
Comment (abhinavdahiya) [inline OWNERS]: rithujohn191 has moved to other things. can you add @ericavonb

Comment (hexfusion) [reply OWNERS]: ack

Comment (hexfusion) [reply OWNERS]: @abhinavdahiya fixed

**Pull request merged**